src/main/java/com/redhat/cloud/notifications/routers/EndpointService.java

```diff
@@ -199,7 +200,7 @@
 
     @PUT
     @Path("/email/subscription/instant")
-    @RolesAllowed("write")
+    @RolesAllowed(RbacIdentityProvider.RBAC_WRITE_INTEGRATIONS)
```

Comment:
How should this be protected? The end-user would initiate this - no? But the end-user only has `integrations:endpoints:read`.
Same below for unsubscribe and the dailyEmail versions.
@theute @josejulio ?

Reply:
I guess email subscriptions should not be protected ? (beside only being able to edit his own settings)

Reply:
Notice we *maybe* don't need this endpoint, unless we want to provide additional flexibility. The user preferences pages uses the [UserConfigService](https://github.com/RedHatInsights/notifications-backend/blob/cdec897fadde7a8e45d130a346b278df2f5abb86/src/main/java/com/redhat/cloud/notifications/routers/UserConfigService.java#L28).

If we want to keep this, maybe only requiring the same permissions as the userpreference page does (only auth required?)